src/types/networks.ts

```diff
@@ -50,6 +50,10 @@ export interface IAdapterNetworkSetting<T = string> {
    */
   testId?: string;
   required?: boolean;
+  /**
+   * Can be used to override or add field validation rules
```

Comment:
I suggest to add a comment that the list of possible keys are available in the `veeValidate` rule definition file.